var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -205,12 +205,18 @@ class PyTorch(PythonPackage, CudaPackage):
 
     @property
     def libs(self):
-        root = join_path(python_platlib, 'torch', 'lib')
+        # TODO: why doesn't `python_platlib` work here?
+        root = join_path(
+            self.prefix, self.spec['python'].package.platlib, 'torch', 'lib'
```

Comment:
Not sure at a first glance on what's going on here. I see `python_platlib` is monkeypatched on all python dependencies. If I have time I'll try to check with a debugger if I can reproduce/spot the issue.

Reply:
Note that the issue occurs when I try to build `py-torchvision`